src/utils/mathUtils.ts

```diff
@@ -61,18 +61,11 @@ export module Utils {
     }
 
     /**
-     * Returns true **only** if x is NaN
-     */
-    export function isNaN(n: any) {
-      return n !== n;
-    }
-
-    /**
      * Returns true if the argument is a number, which is not NaN
      * Numbers represented as strings do not pass this function
      */
     export function isValidNumber(n: any) {
-      return typeof n === "number" && !Plottable.Utils.Math.isNaN(n) && isFinite(n);
+      return typeof n === "number" && n === n && isFinite(n);
```

Comment:
Also there are a number of places in the codebase where people were using `x !== x` instead of the method. While the strict-equal check is concise, I think it can be confusing for people who haven't seen it before.